examples/Analytics.UpdateAction-result.json

```
@@ -0,0 +1,75 @@
+{
+  "@odata.context": "$metadata#SalesModel.SalesData/Sales(Customer(Country),Product(Name),Amount)",
+  "value": [
+    {
+      "Customer": {
+        "Country": "Netherlands"
+      },
+      "Product": {
+        "Name": "Paper"
+      },
+      "Amount": 3,
+      "@Analytics.UpdateAction": {
+        "ActionImport": "UpdateAggregateSales",
+        "Aggregate": "groupby((Customer/Country,Product/Name),aggregate(Amount))/filter(Customer/Country eq 'Netherlands' and Product/Name eq 'Paper')",
```

I'd use something cryptic here to push the point that this is an opaque string.

Examples are easily misinterpreted as specifications.

```suggestion
        "Aggregate": "8C@FA3JWWrFDE@>6C^r@F?ECJ[!C@5F4E^}2>6X[288C682E6Wp>@F?EXX^7:=E6CWrFDE@>6C^r@F?ECJ 6B V}6E96C=2?5DV 2?5 !C@5F4E^}2>6 6B V!2A6CVX",
```